### ISSUE TITLE
IllegalStateException: Speed cannot be 0 for unblocked edge

### ISSUE THREAD

**Issue by hcourt** [bikehopper/graphhopper#89]
```
ERROR i.d.j.e.IllegalStateExceptionMapper - Error handling a request: c404843b2f6de50e
java.lang.IllegalStateException: Speed cannot be 0 for unblocked edge, use access properties to mark edge blocked! Should only occur for shortest path calculation. See #242.
```

Making this to encompass a few issues we've seen:
- #83
- #87

Full stack trace here: https://gist.github.com/graue/2e3dadb9002e0610922aef7fae872651

**Comment by hcourt** [comment]
Oh oops duplicate of #68
